staff/doboapps/js/challenges/scripts/cube.js

```diff
@@ -0,0 +1,18 @@
+function cube(value) {
+
+    var arrayCube = [];
+
+    if (typeof (value) == 'number') {
+        //console.log(Math.pow(value, 3))
+        return Math.pow(value, 3);
+
+    } else if (typeof (value) == 'object') {
+
```

Comment:
TIP Here you can ask if Array.isArray(value), also you could check in the next for if value[i] is a number.

Reply:
wow great!!, thank you!